readthedocs/projects/version_handling.py

```diff
@@ -36,7 +36,7 @@ def prune_major(self, num_latest):
         all_keys = sorted(set(self._state.keys()))
         major_keep = []
         for __ in range(num_latest):
-            if len(all_keys) > 0:
+            if all_keys:
```

Comment:
For this particular case, `all_keys` will always be a sequence, so this logic hasn't changed.

If there is a case where the sequence being tested could be `None`, we should test for that instead of relying on an exception being thrown from `len()`. `if len(sequence)` is also described as an anti-pattern in pep8, so I tend to agree with this sort of change.